modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -49,6 +49,7 @@ import { NodeCallback } from '../types';
 import { Wallet } from '../wallet';
 import { toBitgoRequest } from '../../api';
 import { sanitizeLegacyPath } from '../../bip32path';
+import { PrevOutput } from '../../../../utxo-lib/dist/src/bitgo';
```

Comment:
We can't use relative imports in the SDK like this, since we don't have guarantees that this path is valid at runtime. Please use 
```
import { bitgo } from '@bitgo/utxo-lib';
import PrevOutput = bitgo.PrevOutput;
```

Reply:
Thanks, this was an auto-import but I'll keep this distinction in mind going forward.